setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import find_packages, setup
+
+with open("requirements.in") as f:
+    install_requires = [line for line in f if line and line[0] not in "#-"]
+
+with open("test-requirements.in") as f:
+    tests_require = [line for line in f if line and line[0] not in "#-"]
+
+setup(
+    name="structlog-sentry",
+    version="0.1.0",
```

Comment:
"1.0.0" as it's working and production ready.

Reply:
Do you have any good resources on versioning?

Reply:
https://semver.org

Reply:
and this: https://semver.org/#how-do-i-know-when-to-release-100